website/src/react-native/index.js

```diff
@@ -66,6 +66,9 @@ var App = React.createClass({
           <p>
             All operations between the JavaScript application code and the native platform are performed asynchronously, and the native modules can also make use of additional threads as well.  This means we can decode images off of the main thread, save to disk in the background, measure text and compute layouts without blocking the UI, and more.  As a result, React Native apps are naturally fluid and responsive.  The communication is also fully serializable, which allows us to leverage Chrome Developer Tools to debug the JavaScript while running the complete app, either in the simulator or on a physical device.
           </p>
+          <p>
+            See <a href="docs/debugging.html">Debugging</a>.
+          </p>
           <img src="/react-native/img/chrome_breakpoint.png" width="800" height="443" />
```

Comment:
I added a link to the Debugging doc right after the debugging capability is pitched. I'm not attached to this change, but I think it's a nice touch.